website/site/content/docs/authentication-backends.md

```diff
@@ -74,28 +73,78 @@ can use one of the community-maintained projects below. Feel free to [submit a p
 
 Check each project's documentation for instructions on how to configure it.
```

Comment:
Should the "External OAuth Clients" section be moved below GitLab, or stay here? I'm guessing most people would just use implicit OAuth with GitLab anyway, but there are possible security benefits of using the full flow.

Reply:
Yeah, I was wondering about that, too, or about adding a similar subsection in the `gitlab` backend section. Seeing that the table does include a column for Git host support, I suppose the section really can work generally across hosts/backends, and the setup details are on the individual project repos, so need to worry about that aspect of the content.

OK, done thinking in type. Gonna move it. 😛